### PR TITLE
Add front-end support for roster additions

### PR DESCRIPTION
**Pull request by JBBgameich**
It's working already ... although I'm not satisfied with the desgin. Some things can't be fixed because the same bug as on the loginPage appeared. I would like to rework this soon on a kirigami 2 base.

### CODE REVIEW

File: src/qml/AddContactPage.qml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ *  You should have received a copy of the GNU General Public License
+ *  along with Kaidan. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.0
+import QtQuick.Controls 2.0 as Controls
+import QtQuick.Layouts 1.2
+import org.kde.kirigami 1.0 as Kirigami
+
+Kirigami.Page {
```
Comment (LNJ2) [inline src/qml/AddContactPage.qml]: Don't use a new page! It's just a popup.

File: src/qml/RosterPage.qml [hunk 2]
```diff
@@ -42,14 +43,14 @@ Kirigami.ScrollablePage {
...
-			// 		onTriggered: {
-			// 			//
-			// 		}
-			// 	}
-			// ]
+			actions: [
+				Kirigami.Action {
+					iconName: "bookmark-remove"
+					onTriggered: {
+						kaidan.rosterController.removeContact(model.jid)
```
Comment (LNJ2) [inline src/qml/RosterPage.qml]: `;` is missing

File: src/qml/AddContactPage.qml (hunk 1 above)
Comment (JBBgameich) [inline src/qml/AddContactPage.qml]: I don't want to use the popup anymore, it produces problems with the drawer, is not the best on mobile and has to be centered in the middle of the window. The page looks not so great on desktop yet...

**Comment by LNJ2** [comment]
... no labels :-1:

**Pull request merged**